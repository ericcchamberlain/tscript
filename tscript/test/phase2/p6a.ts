//print = function (x) { console.log(x); };

print(0);

var n;
var j;

n = 5;
j = 10;

x:while (n > 0)
{
  print(1999);
  print(n);
  while (j > 0)
  {
    print(n); print(j);
    if (n + j == 14)
    {
      j = 0;
      continue x;
    }
    j = j - 1;
  }
  print(1066);
  n = n - 1;
}

