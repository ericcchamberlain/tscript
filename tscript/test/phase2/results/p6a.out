0
1999
5
5
10
5
9
1999
5
1066
1999
4
1066
1999
3
1066
1999
2
1066
1999
1
1066
